docs/sophia/data-science/python.md

```diff
@@ -1,85 +1,65 @@
 # Python
 
-We provide prebuilt `conda` environments containing GPU-supported builds of
-`torch`, `tensorflow` (both with `horovod` support for multi-node
-calculations), `jax`, and many other commonly-used Python modules.
+We provide prebuilt `conda` environments containing GPU-supported builds of `torch`, `tensorflow` (both with `horovod` support for multi-node calculations), `jax`, and many other commonly-used Python modules.
 
-Users can activate this environment by first loading the `conda` module, and
-then activating the base environment.
+Users can activate this environment by first loading the `conda` module, and then activating the base environment.
 
 Explicitly (either from an interactive job, or inside a job script):
 
 ```bash
-module use /soft/modulefiles; module load conda ; conda activate base
+module use /soft/modulefiles; module load conda; conda activate base
 ```
 
 This will load and activate the base environment.
 
 ## Virtual environments via `venv`
 
-To install additional packages that are missing from the `base` environment,
-we can build a `venv` on top of it.
+To install additional packages that are missing from the `base` environment, we can build a `venv` on top of it.
 
 !!! success "Conda `base` environment + `venv`"
 
-    If you need a package that is **not** already
-    installed in the `base` environment,
-    this is generally the recommended approach.
+    If you need a package that is **not** already installed in the `base` environment, this is generally the recommended approach.
 
-    We can create a `venv` on top of the base
-    Anaconda environment (with
-    `#!bash  --system-site-packaes` to inherit
-    the `base` packaes):
+    We can create a `venv` on top of the base Anaconda environment (with `--system-site-packages` to inherit the `base` packages):
 
     ```bash
-    module use /soft/modulefiles ; module load conda; conda activate base
+    module use /soft/modulefiles; module load conda; conda activate base
     CONDA_NAME=$(echo ${CONDA_PREFIX} | tr '\/' '\t' | sed -E 's/mconda3|\/base//g' | awk '{print $NF}')
     VENV_DIR="$(pwd)/venvs/${CONDA_NAME}"
     mkdir -p "${VENV_DIR}"
     python -m venv "${VENV_DIR}" --system-site-packages
     source "${VENV_DIR}/bin/activate"
     ```
 
-You can always retroactively change the `#!bash --system-site-packages` flag
-state for this virtual environment by editing `#!bash ${VENV_DIR}/pyvenv.cfg` and
-changing the value of the line `#!bash include-system-site-packages=false`.
+You can always retroactively change the `--system-site-packages` flag state for this virtual environment by editing `${VENV_DIR}/pyvenv.cfg` and changing the value of the line `include-system-site-packages=false`.
```

Comment:
it shouldnt really remove this [InlineHilite](https://facelessuser.github.io/pymdown-extensions/extensions/inlinehilite/) markup...